src/tests/multihost/alltests/conftest.py

```diff
@@ -738,6 +781,38 @@ def remove_nslcd():
     request.addfinalizer(remove_nslcd)
 
 
+@pytest.fixture(scope='class')
+def misc_test(session_multihost, request):
```

Comment:
I would suggest to create multiple fixtures for the different  SSSD proxy configurations in the [domain/...] section. In the example below the SSSD ldap auth provider should be used, so all the `pam_ldap.conf` settings are not needed here, those are needed for the tests where `auth_provider = proxy` and `proxy_pam_target = sssdproxyldap`.

Reply:
changed